File: Initial Prototype/ServerSide/FindNDrive/FindNDriveServices2/DTOs/UserRetrieverDTO.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="UserRetrieverDTO.cs" company="">
//   
// </copyright>
// <summary>
//   Defines the UserRetrieverDTO type.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace FindNDriveServices2.DTOs
{
    using System.Runtime.Serialization;

    /// <summary>
    /// The user retriever dto.
    /// </summary>
    [DataContract]
    public class UserRetrieverDTO
    {
        /// <summary>
        /// Gets or sets the retrieving user id.
        /// </summary>
        [DataMember]
        public int RetrievingUserId { get; set; }

        /// <summary>
        /// Gets or sets the target user id.
        /// </summary>
        [DataMember]
        public int TargetUserId { get; set; }
    }
}

File: Initial Prototype/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/DTOs/UserRetrieverDTO.java
package com.example.myapplication.dtos;

/**
 * Created by Michal on 07/03/14.
 */
public class UserRetrieverDTO {
    private int RetrievingUserId;
    private int TargetUserId;

    public UserRetrieverDTO(int retrievingUserId, int targetUserId) {
        RetrievingUserId = retrievingUserId;
        TargetUserId = targetUserId;
    }
}
